### ISSUE TITLE
datetime.datetime.utcnow() should not have tzinfo

### ISSUE THREAD

**Issue by Pilen** [adamchainz/time-machine#52]
Hi, I find this library very neat and have started using it in multiple tests.
But there is one issue:

```python
In [1]: import datetime

In [2]: import time_machine

In [3]: @time_machine.travel("2020-07-28 13:15:30.478705")
   ...: def foo():
   ...:     return datetime.datetime.utcnow()
   ...:

In [4]: datetime.datetime.utcnow()
Out[4]: datetime.datetime(2020, 7, 31, 13, 46, 25, 828701)

In [5]: foo()
Out[5]: datetime.datetime(2020, 7, 28, 13, 15, 30, 478705, tzinfo=datetime.timezone.utc)
```

time_machine adds a timezone on datetime instances created via `.utcnow()` but this is wrong according to python.
It even specifies it directly in the documentation https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow
> Return the current UTC date and time, **with tzinfo None**.

This change in the behaviour of the datetime library can cause changes in behaviour of the tested code.
Eg. when serialized to a string it adds "+00:00"

**Comment by bbbart** [comment]
I second this. I have a test testing code including `pytz.utc.localize(datetime.utcnow()...)` which now raises a ValueError during time travel, making it impossible to use this library in that particular test.

**Comment by adamchainz** [comment]
Thanks for the report. I've released the fix in version 1.2.1.
